docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,289 @@
+    - Each item in the array MUST be an [artifact descriptor][descriptor], and MUST NOT refer to another `manifest` providing dependency closure.
+    - The max number of blobs is not defined, but MAY be limited by [distribution-spec][oci-distribution-spec] implementations.
+
+- **`subject`** *descriptor*
+
+   An OPTIONAL reference to any existing manifest within the repository. When specified, the artifact is said to be dependent upon the referenced `subject`.
+   - The item MUST be an [artifact descriptor][descriptor] representing a manifest. Descriptors to blobs are not supported. The registry MUST return a `400` response code when `subject` is not found in the same repository, and not a manifest.
+
+- **`annotations`** *string-string map*
+
+    **Pre-Defined Annotation Keys:**
+    - `org.opencontainers.artifact.created` date and time on which the artifact was created (string, date-time as defined by [RFC 3339][rfc-3339])
+
+### Descriptor Properties
+
+Registries and clients work with descriptors as the means to establish discovery and links. To support hashing of different types, enabling filtering, the `artifactType` property is added as an OPTIONAL property to the descriptor: 
+
+```jsonc
+{
+    "digest": "sha256:aaabbb",
+    "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
+    "size": 312,
+    "artifactType": "signature/example"
+}
+```
+
+- **`artifactType`** *string*
+
+  This OPTIONAL property defines the type or Artifact, differentiating artifacts that use the `application/vnd.oci.artifact.manifest`.
+  When the descriptor is used for blobs, this property MUST be empty.
+
+## Registry HTTP API
+
+The `referrers` [extension API](https://github.com/opencontainers/distribution-spec/blob/main/extensions/README.md) is provided to discover these artifacts.
+An artifact client would parse the returned [artifact descriptors][descriptor], determining which  artifact manifest they will pull and process.
+
+The `referrers` API returns all artifacts that have a `subject` of the given manifest digest.
+Reference artifact requests are scoped to a repository, ensuring access rights for the repository can be used as authorization for the referenced artifacts.
+
+### API Path
+
+The `referrers` API is provided on the [distribution-spec][oci-distribution-spec] paths as described below.
+The path of the referrers API  provides consistent namespace/repository paths, enabling registry operators to implement consistent auth access, using existing tokens for content.
+
+**template:**
+
+```rest
+GET /v2/{repository}/_oci/artifacts/referrers?digest={digest}
+```
+
+**expanded example:**
+
+```rest
+GET /v2/hello-world/_oci/artifacts/referrers?digest=sha256:abc123
+```
+
+### Artifact Referrers API results
+
+- Implementations MUST implement [paging](#paging-results).
+- Implementations MUST implement [sorting](#sorting-results)
```

Comment:
I'd like to propose that we make these SHOULDs, to make this easier to adopt for registry implementations.